feature/security/gnsi/credentialz/tests/password_console_login/README.md

```diff
@@ -0,0 +1,44 @@
+# Credentialz-1: Password console login
+
+## Summary
+
+Test that Credentialz properly creates users and the associated password and that the DUT handles
+authentication of those users properly.
+
+
+## Procedure
+
+* Set a username of `testuser` with a password of `i$V5^6IhD*tZ#eg1G@v3xdVZrQwj` using gnsi.Credentialz
+* Perform the following tests and assert the expected result:
+  * Case 1: Success
+    * Authenticate with the `testuser` username and password `i$V5^6IhD*tZ#eg1G@v3xdVZrQwj`
+    * Assert that authentication was successful and some command can be executed. Note: we 
+      simply send a command here rather than assert we "see" a prompt, we do this because this 
```

Comment:
Any suggestions on how the console prompt can be validated? Since the prompt can differ per vendor, would it be preferable to put this assertion under a dut.Vendor flag to "expect" a vendor-specific prompt. Please let me know and I can modify the statement accordingly.

Reply:
I am not familiar with how/if Ondatra says this should be done (I don't primarily work on this) but as I understand the idea is for these tests to run within a KNE environment, where we've got virtualized or containerized vendor images. Depending on how the specific vendor works this matters, but to me that should be invisible to ondatra and some "standard" way of binding to console such as the --serial kvm/qemu binding could make for a standard console-via-telnet/tcp situation. I just don't see that using SSH proves or disproves what we actually wish to assert related to console.

Reply:
Updated this line to reflect the original Readme [statement](https://github.com/openconfig/featureprofiles/tree/main/feature/security/gnsi/credentialz/tests#pass-case). If required, we can change it later once we get ondatra support for making this assertion vendor-neutral (if possible).